packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderButton/ConditionBuilderButton.tsx

```diff
@@ -84,7 +84,7 @@ export const ConditionBuilderButton = ({
   return hideLabel || showToolTip ? (
     <Tooltip
       label={label}
-      align={tooltipAlign}
+      align={tooltipAlign as PopoverAlignment}
```

Comment:
Hey @amal-k-joy, is there a reason we don't type it as `PopoverAlignment` in the `ConditionBuilderButtonProps`?

<img width="519" alt="Screenshot 2024-08-23 at 11 07 33 AM" src="https://github.com/user-attachments/assets/966f455e-727e-41fe-a59b-ac5435dcbfdf">

Reply:
Thanks @AlexanderMelox , Good catch